codacy/values-production.yaml

```diff
@@ -44,6 +44,16 @@ global:
     # consumerPublicKey: ""
     # consumerPrivateKey: ""
 
+  email: {}
```

Comment:
Hah, let's comment this line as well! :smile: 

Or do you prefer to do it when we do the same change for the Git providers?

Reply:
I would prefer to keep it consistent and do that change in all providers when we do this

Reply:
Perfect, sounds good.